grinder/defaultvalues.py

```diff
@@ -106,10 +111,23 @@ class DefaultVulnersScanValues:
     VULNERS_SCRIPT_PATH = "/plugins/vulners.nse"
 
 
+class DefaultMasscanScanValues:
+    """
+    Default values for Masscan scan itself
+    """
+
+    PORTS = None
+    TOP_PORTS = None
```

Comment:
How can I set the `TOP_PORTS` for masscan? CLI? Query file?

Reply:
Now you can specify "top-ports" option in query file like this:
```json
[
    {
        ...
        "masscan_settings": [
            {
                "hosts": "87.250.250.96/27",
                "ports": "",
                "rate": "1000",
                "top-ports": "10"
            }
        ],
        ...
    }
]